backend/apps/volontulo/serializers.py

```diff
@@ -73,11 +90,38 @@ def get_image(self, obj):
             location=image.path.url
         ) if image else None
 
+
+    def create(self, obj):
+        # import pdb; pdb.set_trace()
+        obj["organization_id"] = self.context["request"].data["organization"]["id"]
+        return obj
+       
```

Comment:
Please check empty lines between the methods. 3 lines shouldn't really pass PEP8. Does it pass?